packages/Collapsible/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@paprika/collapsible",
+  "version": "0.1.1",
```

Comment:
Why not `0.1.0`? This is brand new, right?

Reply:
<img width="497" alt="Screen Shot 2019-07-11 at 2 17 07 PM" src="https://user-images.githubusercontent.com/23224777/61086013-9c347580-a3e6-11e9-9154-ffad1fc33b41.png">